docs/guides/shortcuts.md

```diff
@@ -1,37 +1,32 @@
 ---
 layout: default
-title: "Actions"
+title: "Shortcuts"
 lang: en
 ---
 
-# Actions
+# Shortcuts
 
-[Actions](https://api.slack.com/interactivity/actions) are simple shortcuts that people can use to quickly complete a task with your app — like reporting a bug, requesting time off, or starting a meeting.
+Shortcuts are an evolution of slash commands, surfaced in both the composer menu and the quick switcher. Shortcuts allow users to trigger your app's workflows from intuitive surface areas within Slack.
 
-As of March 2020, this SDK supports the following features (yes, there are [more to come soon](https://medium.com/slack-developer-blog/introducing-the-slack-app-toolkit-3d509a15f41b)!).
+For all types of shortcut requests, your app has to respond to the request within 3 seconds by `ack()` method. Otherwise, the user will see the timeout error on Slack.
```

Comment:
Suggestion: Your app has 3 seconds to call `ack()`, which acknowledges a shortcut request is received from Slack.